viewer/packages/camera-manager/src/Flexible/FlexibleCameraManager.ts

```diff
@@ -265,6 +259,16 @@ export class FlexibleCameraManager implements IFlexibleCameraManager {
     this._triggers.controlsTypeChange.subscribe(callback);
   }
 
+  public updateInnerBoundingBox(innerBoundingBox: Box3): void {
+    // If the camera haven't set the position and target before, do it now
+    if (!this.controls.isInitialized) {
```

Comment:
Does this mean we are dependent on the `updateInnerBoundingBox` to be called for the camera manager to be initialized correctly? Not a huge problem, since it's for internal use